_automating-workflows/api/create-workflow.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: default
+title: Create or update a workflow
+parent: Workflow API
+nav_order: 10
+---
+
+# Create or update a workflow
+
+Creating a workflow adds the content of a workflow template to the flow framework system index. You can provide workflows in JSON format (by specifying `Content-Type: application/json`) or YAML format (by specifying `Content-Type: application/yaml`).  By default, the workflow is validated to help identify invalid configurations, including:
+
+* Workflow steps requiring an OpenSearch plugin which is not installed
+* Workflow steps relying on previous node input that is provided by those steps
+* Workflow step fields with invalid values
+* Workflow graph (node/edge) configurations containing cycles or having duplicate IDs
+
+Once a workflow is created, provide its `workflow_id` to other APIs.
+
+The `POST` method creates a new workflow.  The `PUT` method updates an existing workflow.
```

Comment:
The workflow must not have been provisioned for PUT to be successful.  We might want to add that here.